TezosKit/Client/TezosNodeClient.swift

```diff
@@ -280,8 +286,11 @@ public class TezosNodeClient: AbstractClient {
     operationFees: OperationFees? = nil,
     completion: @escaping (Result<String, TezosKitError>) -> Void
   ) {
-    let originateAccountOperation =
-      OriginateAccountOperation(address: managerAddress, contractCode: contractCode, operationFees: operationFees)
+    let originateAccountOperation = operationFactory.originateOperation(
```

Comment:
Splitting account origination and contract origination might make sense, even if they internally call the same functions. As of Athens for example contracts cannot be `spendable`. Also separating out initial storage from contractCode would make the function easier to understand.

Reply:
I deprecated contract origination - it's not going to be a priority for this library given the other options (CLI, ConseilJS) which are properly funded and more appropriate gateways.